### PR TITLE
Create GitHub Pages documentation site with Jekyll

### PR DESCRIPTION
**Pull request by nlakritz**
These changes overhaul the current Github Pages ImageCashLetter documentation site by applying a [customized Jekyll theme](https://github.com/moov-io/bulma-clean-theme) and reorganizing existing documentation.

Project settings under GitHub Pages need to be set to master and the docs folder to publish the site successfully at https://moov-io.github.io/imagecashletter. This new site is currently live at https://nlakritz.github.io/imagecashletter.

### CODE REVIEW

**Review by adamdecaf — APPROVED** [review]

File: docs/_config.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+# You can create any custom variable you would like, and they will be accessible
+# in the templates via {{ site.myvariable }}.
+
+title: Moov ImageCashLetter
+email: security@moov.io
+description: >- # this means to ignore newlines until "baseurl:"
+  Moov ImageCashLetter implements a low-level Image Cash Letter (ICL) interface for parsing, creating, and validating ICL files. Moov ImageCashLetter exposes an HTTP API for REST-based interaction.
+url: "https://moov-io.github.io" # the base hostname & protocol for your site, e.g. http://example.com
+baseurl: "/imagecashletter" # the subpath of your site, e.g. /blog
+twitter_username: jekyllrb
```
Comment (atonks2) [inline docs/_config.yml:27]: Should these values be Moov specific?

Comment (nlakritz) [reply docs/_config.yml:27]: `twitter_username` and `github_username` are actually for the default template, so they don't do anything for us. I'll just remove them.

Comment (atonks2) [reply docs/_config.yml]: Oh ok. Sounds good.

**Review by atonks2 — APPROVED** [review]
This is great! Thanks for putting this together.

**Pull request merged**